httpserver/http_connection.cpp

```diff
@@ -592,6 +594,62 @@ void connection_t::handle_wrong_swarm(const std::string& pubKey) {
     BOOST_LOG_TRIVIAL(info) << "Client request for different swarm received";
 }
 
+constexpr auto DB_POLL_FREQUENCY = std::chrono::milliseconds(200);
```

Comment:
Technically this is a period not a frequency :)

Reply:
Yeah, I thought about it for a bit and couldn't remember what the "opposite" of frequency is :)

Reply:
fixed